README.md

```diff
@@ -627,6 +627,10 @@ Returns a function that will set a lens to `false`
 `value -> arrayLens -> includeLens`
 An include lens represents membership of a value in a set. It takes a value and lens and returns a new lens - kind of like a "writeable computed" from MobX or Knockout. The view and set functions allow you to read and write a boolean value for whether or not a value is in an array. If you change to true or false, it will set the underlying array lens with a new array either without the value or with it pushed at the end.
 
+#### optionLens
+`value -> arrayLens -> optionLens`
```

Comment:
Shouldn't this be `value -> lens -> optionLens`?

Reply:
it's not a dom lens, it's a higher-order lens - it belongs in the same category as `includeLens`, since it's the same kind of thing.

And yeah, you're right. I just copied the signature from `includeLens`, but I guess it's wrong for both 🤔

Reply:
(the difference, to clarify, is that the higher-order lenses `includeLens` and `optionLens` take a lens and return a lens, while the domLens methods take a lens and return an object mapping DOM properties to functions that manipulate the lens)